### PR TITLE
[New Rule] UAC Bypass via Elevated COM IEinstall

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/449

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
Example of Alert:

image

File: rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via abusing an elevated COM Interface to launch a malicious program.
+Attackers bypass UAC to stealthily execute code with elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Bypass UAC via Elevated COM Internet Explorer Add-on Installer"
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml:17]:
```suggestion
name = "UAC Bypass Attempt via Elevated COM Internet Explorer Add-On Installer"
```

File: rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via abusing an elevated COM Interface to launch a malicious program.
+Attackers bypass UAC to stealthily execute code with elevated permissions.
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml:11]:
```suggestion
Identifies User Account Control (UAC) bypass attempts by abusing an elevated COM Interface to launch a malicious
program. Attackers may attempt to bypass UAC to stealthily execute code with elevated permissions.
```

File: rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+risk_score = 47
+rule_id = "fc7c0fa4-8f03-4b3e-8336-c5feab0be022"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started", "info") and
+ wildcard(process.executable, "C:\\*\\AppData\\*\\Temp\\IDC*.tmp\\*.exe") and
+  process.parent.name == "ieinstal.exe" and process.parent.args == "-Embedding"
```
Comment (threat-punter) [inline rules/windows/privilege_escalation_uac_bypass_com_ieinstal.toml:28]:
```suggestion
 process.parent.name == "ieinstal.exe" and process.parent.args == "-Embedding"
```

**Review by threat-punter — APPROVED** [review]

**Review by bm11100 — APPROVED** [review]
🥇

**Pull request merged**